src/styles/blocks/card.scss

```diff
@@ -0,0 +1,122 @@
+.card {
+  width: 200px;
+  background-color: $white;
+  border: 1px solid $grey;
+  border-radius: 5px;
+
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+  margin-right: 48px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent;
+    margin: 0;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin: 4px 0 16px;
+  }
+
+  &__review-wrapper {
+    @include flex-justify-align(space-between, center);
+  }
+
+  .stars {
+    display: flex;
+
+    &__star {
+      width: 16px;
+      height: 16px;
+      margin-right: 4px;
+      background-image: url("/images/star.svg");
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+
+    &__star:last-child {
+      margin-right: 0;
+    }
+  }
+
+  .stars--1 .stars__star:nth-child(-n+1),
+  .stars--2 .stars__star:nth-child(-n+2),
+  .stars--3 .stars__star:nth-child(-n+3),
+  .stars--4 .stars__star:nth-child(-n+4),
+  .stars--5 .stars__star:nth-child(-n+5) {
+    background-image: url("/images/star-active.svg");
+  }
```

Comment:
let's pursue the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md#:~:text=Try%20using%20for%20loops%20for%20the%20stars%20selector.)
<img width="632" alt="loop" src="https://user-images.githubusercontent.com/39308717/200796670-df5e57b0-267e-408c-8595-6e80c7775c5c.png">

Reply:
Thank all and @meforoma a special thank you for explaining visually.

Keep "pounding" by showing me my mistakes, because I'm here to learn and to get better at FE.